src/rest.rs

```diff
@@ -6,28 +6,41 @@ use regex::Regex;
 use serde::{Deserialize, Serialize};
 use serde_repr::{Deserialize_repr, Serialize_repr};
 
+use crate::auth::Auth;
 use crate::error::*;
 use crate::options::ClientOptions;
-use crate::{auth, crypto, history, http, json, presence, stats, Result};
+use crate::{crypto, history, http, json, presence, stats, Result};
 
 /// A client for the [Ably REST API].
 ///
 /// [Ably REST API]: https://ably.com/documentation/rest-api
 #[derive(Debug)]
 pub struct Rest {
-    pub auth: auth::Auth,
-    pub channels: Channels,
-    pub client: Client,
+    pub channels: (),
```

Comment:
Is this needed?

Reply:
So I wasn't sure if there was going to be a need for some sort of channels object (HashMap<String, Channel> or something). The spec points out the channel.get function should return existing channels https://ably.com/docs/client-lib-development-guide/features#RSN2. So I'm assuming there is need for a channel field but I just don't know the type yet.

Reply:
Ok thanks. For the record, I dislike the `channels` object from the spec 🙂